### DISCUSSION TITLE
[ FF8 2000 ] No-CD fix or hext

### DISCUSSION THREAD

**Discussion by shikulja** [julianxhokaxhiu/FFNx#194] [General]
Needed in order to keep files intact, 
and nocd old ones under windows 7+ do not work

**Comment by julianxhokaxhiu** [comment]
Considering the fact that the fix is enabled and working ONLY for Steam release in order to comply with Legal Terms, even implementing that during DLL runtime loading on FF8 would definitely not work anyway.

Also, just to be clear this will NEVER be implemented in order to avoid piracy on the game. If you want a No-CD experience, buy the Steam release :)

Cheers!

**Comment by shikulja** [comment]
@julianxhokaxhiu 
Most games for the final fantasy series were bought, 
![Аннотация 2020-06-01 214756](https://user-images.githubusercontent.com/3386359/83442941-a1ad0b00-a451-11ea-9b3e-467fc5216f00.png)

but there are problems from the legal side, especially for steam.
but you can add to the classic pc-2000, considering that you have separate dlls for it.
and you need it purely for the convenience of those who play the classic version.
by the way, the gamepad doesn’t work on the classic version, support was written in readme,
but for this not. Is this by chance not due to the fact that only Steam versions of games are in priority?

**Comment by shikulja** [comment]
@julianxhokaxhiu 
as far as I know that if the program code (exe) is not modified, this is not a violation of the agreements.

most modern computers do not have a cd drive, even to get(dump) the image of a purchased licensed game is a problem.
This will solve the issue of usability of classic versions,
Considering the fact that exe themselves will not be modified. therefore, this is fix and not Patch.
it is possible to add logic to enable this fix in ffnx only if the necessary archives are located in the game directory, how they were received is up to each user, not the developers.

even gog practices the use of nocd patches for old games, as it is justified in modern realities. If this is not a patch but a fix, I think it should be normal, on the legal side, and has a mediocre attitude to piracy, since whoever wants to can do it without ffnx)
